tests/flot.scattergraph.Test.js

```diff
@@ -533,4 +533,22 @@ describe('A scatter graph', function () {
         });
         expect(index).toEqual(2);
     });
+    it('should clip points on axis min and max', function() {
+        var plot = $.plot(placeholder, [[]], options);
+
+        series.data = [[{x: 1, y: 2}, {x: 2, y: 3}, {x: 3, y: 4}, {x: 4, y: 5}, {x: 5, y: 6}]];
+
+        spyOn(ctx, 'rect').and.callThrough();
+        spyOn(ctx, 'clip').and.callThrough();
+
+        plot.hooks.processRawData.forEach(function (hook) {
+            hook(plot, {points: [], xaxis: {options: {}}, yaxis: {options: {}}}, series.data, {points: []});
+        });
+        plot.hooks.drawSeries.forEach(function (hook) {
+            hook(plot, ctx, series);
+        });
+
+        expect(ctx.rect).toHaveBeenCalled();
```

Comment:
The most interesting test would be to actually verify we called clip with the correct values. If that's too much a pain, then I'm fine with this.